src/components/dashboard/main-content/course-enrollments/course-cards/BaseCourseCard.jsx

```diff
@@ -160,7 +161,18 @@ const BaseCourseCard = ({
 
   const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
 
-  const CourseTitleComponent = externalCourseLink ? Hyperlink : Link;
+  const renderCourseTitle = () => {
+    const courseTitleComponentProps = {
+      className: classNames('h3 mb-0', { 'text-white': isExecutiveEducation2UCourse }),
+    };
+    if (type === COURSE_STATUSES.upcoming) {
+      return <Card.Header title={title} {...courseTitleComponentProps} />;
```

Comment:
[consideration] I wonder if it makes sense to just use a `div` here since the dashboard course cards are not actually using Paragon's `Card` component. It feels a bit odd to use `Card.Header` outside of the context of `Card` itself.

```jsx
return <div {...courseTitleComponentProps}>{title}</div>;
```